### ISSUE TITLE
1.79.0-insiders only shows single task - even when there are multiple

### ISSUE THREAD

**Issue by ThaDaVos** [actboy168/vscode-tasks#44]
Since my editor updated to the latest insiders (as I have a few extensions which preview is only available on Insiders) - I only see a single task in the taskbar.

My tasks.json:
```JSON
{
    "version": "2.0.0",
    "tasks": [
        {
            "label": "publish",
            "command": "dotnet",
            "type": "process",
            "args": [
              "publish",
              "${workspaceFolder}/RDPBootupConfigurator.csproj",
              "/property:GenerateFullPaths=true",
              "/consoleloggerparameters:NoSummary",
              "-c",
              "Release",
              "-o",
              "${workspaceFolder}/out/publish"
            ],
            "problemMatcher": "$msCompile"
        },
        {
            "label": "build",
            "command": "dotnet",
            "type": "process",
            "args": [
                "build",
                "${workspaceFolder}/RDPBootupConfigurator.csproj",
                "/property:GenerateFullPaths=true",
                "/consoleloggerparameters:NoSummary"
            ],
            "problemMatcher": "$msCompile"
        }
    ]
}
```

If I bootup my normal vscode which is 1.78.2 - it shows them without issues

**Comment by actboy168** [comment]
You should report it to VSCode team.

**Comment by jrieken** [comment]
@actboy168 you are correct in that we make changes here but this exposes an issue how the `createStatusBarItem`-API is being used. In https://github.com/actboy168/vscode-tasks/blob/master/extension.js#L365 multiple items with the _same_ identifier are being created which isn't how the API meant to be used. The point of the identifier is to be unique within the extension.

**Comment by jrieken** [comment]
> the statusbar with the same id will be considered as the same class, and it can be disabled or enabled uniformly in the right-click menu. Now we have lost it.

Looked at this again and you can get this behaviour by not using an identifier at all, then they will be all treated equal and hiding one will hide them call. The, be aware that this isn't how you we like status bar item be used

**Comment by actboy168** [comment]
> Looked at this again and you can get this behaviour by not using an identifier at all, then they will be all treated equal and hiding one will hide them call. The, be aware that this isn't how you we like status bar item be used

OK, as long as it works. I don't mind doing more things that aren't pretty.
